app/services/digest_initiator_service.rb

```diff
@@ -16,7 +18,14 @@ def call
       digest_run.save!
     end
 
-    #enqueue the digest creation workers TBC
+    #TODO think about this being retried
+    #
+    subscribers = DigestRunSubscriberQuery.call(digest_run: digest_run)
+
+    subscribers.each do |subscriber|
+      digest_run_subscriber = DigestRunSubscriber.create(digest_run: digest_run, subscriber: subscriber)
```

Comment:
Don't know if it matters at this point, but we may want to insert these in bulk and then call the worker using the `.ids` field of the result value from `DigestRunSubscriber.import!`.

Reply:
This might be a bit of a premature optimisation but we've used the same pattern a lot so I've updated in 6e30b26.